src/lib/syncPagePath.ts

```diff
@@ -238,7 +238,12 @@ export default async function syncPagePath(this: APIv2PageUploadCommands) {
     );
   }
 
-  let unsortedFiles = await findPages.call(this, pathInput);
+  const validFileExtensions = ['.markdown', '.md', '.mdx'];
```

Comment:
could we instead reuse [this array](https://github.com/readmeio/rdme/blob/5c1567eb8d1155675ef7645bcce54e8f8c2e191c/src/lib/readPage.ts#L83) (i.e., create an array and import it in a few places) and import it here?

Reply:
perfect, tysm!